projects/js-packages/charts/src/components/line-chart/stories/index.docs.mdx

```diff
@@ -0,0 +1,593 @@
+
+<Source
+	language="jsx"
+	code={ `<LineChart
+		data={data}
+		onPointerDown={({ datum }) => console.log('Clicked:', datum)}
+		onPointerMove={({ datum }) => console.log('Hover:', datum)}
+	/>` }
+/>
+
+### Keyboard Navigation
+
+Charts are fully keyboard accessible with Tab navigation and arrow key data point selection.
+
+## Legends
+
+### Basic Legend
+
+Display series information with automatic color matching:
+
+<Canvas of={ LegendStories.Default } />
+
+<Source
+	language="jsx"
+	code={ `<LineChart
+		data={data}
+		showLegend={true}
+		legendOrientation="horizontal"
+		legendAlignmentHorizontal="center"
+		legendAlignmentVertical="bottom"
+	/>` }
+/>
+
+### Custom Legend Shapes
+
+Use different shapes and custom glyphs in legends:
+
+<Canvas of={ LegendStories.AlignmentPositioning } />
+
+	<Source
+		language="jsx"
+		code={ `<LineChart
+			data={data}
+			showLegend={true}
+			legendAlignmentHorizontal="right"
+			legendAlignmentVertical="top"
+		/>` }
+	/>
+
+## Advanced Customization
+
+### Axis Configuration
+
+Customize axes with formatting, orientation, and tick options:
+
+<Canvas of={ LineChartStories.SmartFormatting } />
+
+<Source
+	language="jsx"
+	code={ `<LineChart
+		data={data}
+		options={{
+			axis: {
+				x: {
+					orientation: 'bottom',
+					numTicks: 6,
+					tickFormat: (date) => date.toLocaleDateString()
+				},
+				y: {
+					orientation: 'right',
+					numTicks: 4,
+					tickFormat: (value) => '$' + value + 'k'
+				}
+			},
+			yScale: {
+				type: 'linear',
+				zero: true,
+				nice: true
+			}
+		}}
+	/>` }
+/>
+
+### Responsive Behavior
+
+Charts automatically resize based on container size:
+
+<Canvas of={ LineChartStories.FixedDimensions } />
+
+<Source
+	language="jsx"
+	code={ `// Responsive (default)
+	<LineChart data={data} />
+
+	// Fixed dimensions
+	<LineChart
+		data={data}
+		width={800}
+		height={400}
+	/>` }
+/>
+
+### Custom Margins
+
+Control chart layout with precise margin settings:
+
+<Source
+	language="jsx"
+	code={ `<LineChart
+		data={data}
+		margin={{
+			top: 20,
+			right: 80,
+			bottom: 60,
+			left: 80
+		}}
+	/>` }
+/>
+
+## Annotations
+
+Add contextual information to highlight specific data points or events. For comprehensive annotation documentation, see the [Line Chart Annotations](?path=/docs/js-packages-charts-types-line-chart-annotations--docs) guide:
+
+<Source
+	language="jsx"
+	code={ `<LineChart data={data}>
+		<LineChart.AnnotationsOverlay>
+			<LineChart.Annotation
+				datum={{ date: new Date('2024-01-15'), value: 100 }}
+				title="Product Launch"
+				subtitle="Version 2.0 released"
+				subjectType="line-vertical"
+			/>
+		</LineChart.AnnotationsOverlay>
+	</LineChart>` }
+/>
+
+## Data Formats
+
+### Date Handling
+
+The chart supports multiple date formats and automatically converts to local timezone:
+
+<Canvas of={ LineChartStories.DateStringFormats } />
+
+<Source
+	language="jsx"
+	code={ `// Date objects (recommended)
```

Comment:
```suggestion
	code={ `// Date objects
```

I wouldn't recommend to use date objects because consumers have to construct them in their code from strings as well. They could, but I don't think it's recommended.